_data/changelogs/docs-settings.yml

```diff
@@ -0,0 +1,296 @@
+    affectsGuidance:
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles: true
+    githubPr: 4726
+    githubRepo: uswds
+    versionUswds: 3.0.2
+  - date: 2022-04-28
+    summary: Replaced `$output-all-utilities` with `$output-these-utilities`.
+    summaryAdditional: This setting allows users to create a list of the utility modules they want to include in the project.
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance:
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles: true
+    githubPr: 4631
+    githubRepo: uswds
+    versionUswds: 3.0.0
+  - date: 2022-04-28
+    summary: Updated the settings page to use USWDS 3.0 conventions.
+    summaryAdditional:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1538
+    githubRepo: uswds-site
+  - date: 2022-03-11
+    summary: Updated value of `$theme-modal-lg-max-width` from `800px` to `880px`.
+    summaryAdditional:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1474
+    githubRepo: uswds-site
+  - date: 2021-06-16
+    summary: Updated input settings.
+    summaryAdditional: Added `$theme-input-background-color`; removed `$theme-input-tile-background-color-selected`, `$theme-input-tile-border-color`, and `$theme-input-tile-border-color-selected`.
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance:
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles: true
+    githubPr: 4199
+    githubRepo: uswds
+    versionUswds: 2.12.0
+  - date: 2021-06-16
+    summary: Added pagination settings.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance:
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles: true
+    githubPr: 4175
+    githubRepo: uswds
+    versionUswds: 2.12.0
+  - date: 2021-06-16
+    summary: Updated values for `$theme-color-success-dark` and `$theme-color-success-darker`.
+    summaryAdditional:
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance:
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles: true
+    githubPr: 4183
+    githubRepo: uswds
+    versionUswds: 2.12.0
+  - date: 2021-04-28
+    summary: Updated our docs and settings to remove `$theme-site-max-width`.
+    summaryAdditional: In its place, use `$theme-grid-container-max-width`.
+    isBreaking:
+    affectsAccessibility:
+    affectsAssets:
+    affectsGuidance: true
+    affectsJavascript:
+    affectsMarkup:
+    affectsStyles:
+    githubPr: 1198
+    githubRepo: uswds-site
+    versionUswds:
+  - date: 2021-03-17
+    summary: Added `$theme-prefix-separator`.
+    summaryAdditional: This setting allows you to output any character or string for the separator between the prefix and main class.
```

Comment:
It's not major, but I'm starting to see a little discrepancy between when summaryAdditional is used. In this example, it's detail followed by more detail. In some examples, like "updated page for clarity," there's less detail for less detail--summaryAdditional isn't used at all in those cases. It might be worth updating the readme to address when to include more detail and how.

Reply:
I see an inconsistency too. Part of me wonders if we need the level of detail we are giving. I might open up an issue so I can step back and evaluate. Are you comfortable moving forward as-is for now?

Reply:
Yes! Good idea.